create-icecc-env-linux.py

```diff
@@ -71,6 +71,16 @@ def DownloadUrl(url, output_path):
         time.sleep(retry_wait_s)
         retry_wait_s *= 2
 
+def MovePlugin(plugin_base_name, clang_prefix_path):
+    plugin_src_name = plugin_base_name + ".so"
```

Comment:
If we want this to work correctly on Linux as well (eventually) should we add it both as .so and as .dylib to the package?

Reply:
OK.